<?xml version="1.0"?>
<launch>

  <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}"/>

  <!--  sensor manager -->
  <node pkg="ariac_group1" 
        type="ariac_group1_sensor_manager_node" 
        name="sensor_manager" output="screen"></node>


  <!--  factory manager -->
  <node pkg="ariac_group1" 
        type="ariac_group1_rwa2_node" 
        name="rwa4" output="screen"></node>

</launch>
